sympy/parsing/tests/test_latex_lark.py

```diff
@@ -178,13 +178,11 @@ def _log(a, b=E):
     (r"\int^{b}_{a} x dx", Integral(_Mul(1, x), (x, a, b))),
     (r"\int_{f(a)}^{f(b)} f(z) dz", Integral(f(z), (z, f(a), f(b)))),
     (r"\int a + b + c dx", Integral(_Mul(1, _Add(_Add(a, b), c)), x)),
-    (r"\int \frac{dz}{z}", Integral(Pow(z, -1), z)),
-    (r"\int \frac{3 dz}{z}", Integral(3 * Pow(z, -1), z)),
+    (r"\int \frac{dz}{z}", Integral(_Mul(1, _Mul(1, Pow(z, -1))), z)),
+    (r"\int \frac{3 dz}{z}", Integral(_Mul(1, _Mul(3, _Pow(z, -1))), z)),
```

Comment:
Why does the extra 1 appears in multiplication?

Reply:
I'm not entirely sure. This is some oddity which is manifesting itself when the parser interacts with the `evaluate(False)` context manager. Perhaps you can mention this in #25620 as well.

Since it's just random multiplications by 1, I'm not too worried about it as the correctness is not affected.